scss/components/_video-card.scss

```diff
@@ -0,0 +1,36 @@
+.video-card {
+    picture {
+        cursor: pointer;
+
+        img {
```

Comment:
img {} can stay outside of picture, we won't have other images in video-card object